build-tools/README.md

```diff
@@ -16,6 +17,11 @@ policy-check, fluid-bump-version, etc.
 Note: Don't add new CLI commands to this package. Instead, add a new command to the `build-cli` package and import the
 functionality you need from this package.
 
+## @fluid-tools/version-tools
+
+This package contains APIs and a CLI for working with semantic versioning version strings and ranges, especially those
+using [Fluid-specific version schemes.](./packages/version-tools/README.md#version-schemes)
```

Comment:
(Not an issue for this PR) Do we currently do link validation on links in READMEs like this? Or do we just validate the website currently?

Reply:
Only the website, but there is a markdownlint config file in the root where I started to set some linting up a few years ago. Also I don't think the linting in the docs folder itself is run in CI. :( #9978

Reply:
Good to know. I'm collecting misc. items for improving docs directory reliability for the larger website refactoring effort. I'll reference that there.

Reply:
Added an ADO item that points to the above issue.
https://dev.azure.com/fluidframework/internal/_workitems/edit/1366